x-pack/plugin/security/qa/profile/build.gradle

```diff
@@ -23,6 +26,13 @@ testClusters.matching { it.name == 'javaRestTest' }.configureEach {
 
   setting 'xpack.security.authc.realms.ldap.ldap1.enabled', 'false'
   setting 'xpack.security.authc.domains.my_domain.realms', 'default_file,ldap1'
+
+  // randomize between hashed and literal username for profile uid
+  if (literalUsername) {
+    setting 'xpack.security.authc.domains.my_domain.uid_generation.literal_username', 'true'
+    setting 'xpack.security.authc.domains.my_domain.uid_generation.suffix', 'es'
+  }
```

Comment:
This just test the new settings. How the settings work is tested with `ProfileUidIntegTests`.